man/systemd-cryptenroll.xml

```diff
@@ -401,6 +403,31 @@
         <xi:include href="version-info.xml" xpointer="v248"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--fido2-salt-file=<replaceable>PATH</replaceable></option></term>
+
+        <listitem><para>When enrolling a FIDO2 security token, specifies the path to a file or an
+        <constant>AF_UNIX</constant> socket from which we should read the salt value to be used in the
+        HMAC operation performed by the FIDO2 security token. If this option is not specified, the salt
+        will be randomly generated.</para>
+
+        <xi:include href="version-info.xml" xpointer="v257"/></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><option>--fido2-token-parameters=<replaceable>BOOL</replaceable></option></term>
+
+        <listitem><para>When enrolling a FIDO2 security token, controls whether to store FIDO2
+        parameters in a LUKS2 token. Defaults to <literal>true</literal>, i.e. by default a LUKS2
+        token is created and FIDO2 parameters are stored in it. If set to <literal>false</literal>,
```

Comment:
well, the token is not "created" i'd say. we store the data in the superblock inside a LUKS2 token structure.

Reply:
Removed this whole sentence, it's redundant anyway.